ropetest/type_hinting_test.py

```diff
@@ -161,6 +166,105 @@ def test_hint_attr_for_post_defined_type(self):
         result = self._assist(code, offset)
         self.assert_completion_in_result('isAlive', 'attribute', result)
 
+    def test_hint_parametrized_list(self):
+        code = 'class Sample(object):\n' \
```

Comment:
I'm not sure with this proposal.
1. It increases coupling of some methods and decreases cohesion of class.
2. It degrades readability, in case test fails it complicates the understanding and navigation in the code.
3. It's contrary for one of the main purpose of tests - demonstrate case of usage. The demonstration should be as easy as possible.
4. I can't apply this pattern for test_hint_parametrized_nested_tuple_list(), test_hint_parametrized_dict_key() and test_hint_parametrized_dict_value(). So, there is only partial overlap, not systematical.